lib/connection/result/column.js

```diff
@@ -550,7 +551,18 @@ function convertRawVariant(rawColumnValue, column, context)
   {
     try
     {
-      ret = eval("(" + rawColumnValue + ")");
+      // check if raw string is in XML format
+      // ensure each tag is enclosed and all attributes and elements are valid
+      if (XMLValidator.validate(rawColumnValue) === true)
```

Comment:
Please run some perf tests with previous release of the driver and make sure this won't hurt the performance.

Reply:
The execution time of convertRawVariant() increases by ~1.2 ms compared to previous version

Reply:
Would it be better to parse the string using the JSON parser first, if it fails then try parsing it as an XML?

This would keep performance the same for JSON strings and fix the issue for XML strings

Reply:
Sure.